.github/olm-publish/action.yaml

```diff
@@ -33,6 +35,18 @@ runs:
       username: ${{ inputs.quay_login }}
       password: ${{ inputs.quay_token }}
 
+  - name: Git merge olm-catalog branch
+    shell: bash
+    run: |
+      git config user.name rhobs-release-bot
+      git config user.email release-bot@monitoring.rhobs.io
+
+      # NOTE: uses strategy-option=ours as bundle created for a release using
+      # make initiate-release conflicts with the previous bundle created for
+      # development branch, so the version in main takes precedence over the
+      # one in olm-catalog branch
+      git merge -Xours origin/olm-catalog  -m "merge olm-catalog branch"
```

Comment:
Does the merge message need to conform to the commit linter expectations?

Reply:
Good catch, fixed 🤞